src/NewsService.Models.Dto/Responses/NewsResponse.cs

```diff
@@ -1,15 +1,16 @@
-﻿using LT.DigitalOffice.NewsService.Models.Dto.Models;
-using System;
+﻿using System;
+using LT.DigitalOffice.NewsService.Models.Dto.Models;
 
 namespace LT.DigitalOffice.NewsService.Models.Dto.Responses
 {
-    public class NewsResponse
-    {
-        public Guid Id { get; set; }
-        public string Content { get; set; }
-        public string Subject { get; set; }
-        public User Author { get; set; }
-        public Department Department { get; set; }
-        public bool IsActive { get; set; }
-    }
+  public record NewsResponse
```

Comment:
you have news too, for what this models?

Reply:
For get request